_posts/2022-05-18-torchopt/torchopt.Rmd

```diff
@@ -0,0 +1,356 @@
+---
+title: "Community spotlight: Fun with torchopt"
+description: >
+  Today, we want to call attention to a highly useful package in the torch ecosystem: torchopt. It extends torch by providing a set of popular optimization algorithms not available in the base library. As this post hopes to show, it is also fun to use!
+author:
+  - name: Sigrid Keydana
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+slug: keydanatorchoptim
+date: 05-18-2022
+categories:
+  - Torch
+  - R
+  - Packages/Releases
+bibliography: bibliography.bib
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+preview: images/petals.png
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE, fig.width = 6, fig.height = 6)
+```
+
+Pretty much from the beginnings, it has been exciting to watch the growing number of packages developing in the `torch` ecosystem. Well, not just the number. What's amazing is the variety of things people do with `torch`: extend its functionality; integrate and put to domain-specific use its low-level automatic differentiation infrastructure; port neural network architectures ... and last but not least, answer scientific questions.
+
+This blog post will introduce, in short and rather subjective form, one of these packages: [`torchopt`](https://github.com/e-sensing/torchopt). Before we start, one thing we should probably say a lot more often: If you'd like to publish a post on this blog, on the package you're developing or the way you employ R-language deep learning frameworks, let us know -- you're more than welcome![^1]
```

Comment:
Hm yeah, I see ... I'll probably leave it there  anyway - instead of moving to the end - just to have it in focus _once_ ... but will move the link then , as you say